frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -64,12 +64,19 @@ const csvName = () =>
     );
 
 const getActions = (selectedObj: any) => {
+  let group = '';
+
+  if (selectedObj.apiVersion.includes('/')) {
+    group = groupVersionFor(selectedObj.apiVersion).group;
+  } else {
+    group = selectedObj.apiGroup;
```

Comment:
What is `selectedObj`? k8s resources don't have this field.

Reply:
I have passed the `kindObj<k8sKind>` from [here](https://github.com/openshift/console/pull/4352/files#diff-bc19a445cd55d99705e404fae2c786edR598) which does not have `apiVersion` as a property but have `apiGroup` in order to get the group.